src/helpers/translations.helper.ts

```diff
@@ -1,6 +1,11 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import i18n from "i18next";
+import translations from "../assets/translations";
+export function t(r: keyof typeof translations["pl"]["translation"], options?: {}): string {
```

Comment:
Można by było wynieść to do osobnego typu, ponieważ ten typ może być dość często używany.
Faktycznie powinien być przy każdym tłumaczeniu, żeby było wiadomo jakie klucze należy zdefiniować w tym pliku.

Czyli zrobić coś w stylu:
```
type Translation = keyof typeof translations["pl"]["translation"]
```

Reply:
No i też dodać ten typ przy tłumaczeniu angielskim

Reply:
?